adapter/config/types.go

```diff
@@ -325,6 +327,17 @@ type analytics struct {
 	Enforcer analyticsEnforcer
 }
 
+type tracing struct {
```

Comment:
formatting ?

Reply:
if you are using VS code, the IDE does the formatting by itself when you save the file.

Reply:
Fixed in 8d52822bdbad80f19fc2738c59f60af36e75e5c8